### PR TITLE
Inquiry and InquiryItems part

### PR DESCRIPTION
**Pull request by vbartusevicius**
Summary: Inquiry + InquiryItem Mappers

### CODE REVIEW

File: src/Paysera/WalletApi/Mapper.php [hunk 1]
```diff
@@ -1959,4 +1969,73 @@ protected function createDateTimeFromTimestamp($timestamp)
...
+        return array_filter($data);
+    }
+
+    public function encodeInquiry(Paysera_WalletApi_Entity_Inquiry_Inquiry $inquiry)
+    {
+        $data = array(
+            'key' => $inquiry->getIdentifier(),
+            'type' => $inquiry->getType(),
+            'description' => $inquiry->getDescription(),
+            'status' => $inquiry->getStatus(),
```
Comment (mariusbalcytis) [inline src/Paysera/WalletApi/Mapper.php]: We should not pass this as this is read-only.

File: src/Paysera/WalletApi/Mapper.php [hunk 2]
```diff
@@ -1959,4 +1969,73 @@ protected function createDateTimeFromTimestamp($timestamp)
...
     }
-}
\ No newline at end of file
+
+    public function decodeInquiry(array $inquiryData)
+    {
+        $inquiry = new Paysera_WalletApi_Entity_Inquiry_Inquiry();
+
+        if (isset($inquiryData['key'])) {
+            $inquiry->setIdentifier($inquiryData['key']);
```
Comment (mariusbalcytis) [inline src/Paysera/WalletApi/Mapper.php]: Wouldn't it be more clear to keep the naming intact?

File: src/Paysera/WalletApi/Mapper.php [hunk 3]
```diff
@@ -691,6 +691,11 @@ public function encodeTransaction(Paysera_WalletApi_Entity_Transaction $transact
         if ($transaction->getUserInformation() !== null) {
             $result['user'] = $this->encodeUserInformation($transaction->getUserInformation());
         }
+        if ($transaction->getInquiries() !== null && count($transaction->getInquiries()) > 0) {
```
Comment (mariusbalcytis) [inline src/Paysera/WalletApi/Mapper.php]: No point in checking if `null`, as it should never be so

File: src/Paysera/WalletApi/Entity/Inquiry/InquiryItem.php [hunk 4]
```diff
@@ -0,0 +1,63 @@
+<?php
+
+
+class Paysera_WalletApi_Entity_Inquiry_InquiryItem
+{
```
Comment (mariusbalcytis) [inline src/Paysera/WalletApi/Entity/Inquiry/InquiryItem.php]: Maybe we should provide available types for items?

By the way, title will not be used yet with current available type.

**Comment by vbartusevicius** [comment]
Added requested changes

**Pull request merged**